### PR TITLE
SD-3067 Publish stories asynchronously

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by akintolga** [comment]
I think we better have this in a separate branch on its own while we review the functionality once the wip is removed from this PR. So we and QA can check the publishing functionality before merging.

File: server/apps/publish/content/common.py [hunk 1]
```diff
@@ -86,61 +86,13 @@ class BasePublishService(BaseService):
...
-        self._validate_package_contents(original, takes_package, package_validation_errors)
-        if len(package_validation_errors) > 0:
-            raise ValidationError(package_validation_errors)
-
+        updates['digital_item_id'] = self._get_digital_id_for_package_item(original)
+        self._validate(original, updates)
         self._set_updates(original, updates, updates.get(config.LAST_UPDATED, utcnow()))
-        updates[ITEM_OPERATION] = ITEM_PUBLISH
-        convert_task_attributes_to_objectId(updates)
+        convert_task_attributes_to_objectId(updates) # ???
```
Comment (mdhaman) [inline server/apps/publish/content/common.py]: Problem is how the task attributes are stored. It is stored as string or `ObjectId` and for mongo queries to work we need all as `ObjectId`.  Other option might be to have schema for the `task`?

**Comment by ioanpocol** [comment]
Please rebase

**Comment by mugurrus** [comment]
Made the final PR on superdesk-core

**Pull request closed without merging**